content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst

```diff
@@ -1,87 +1,65 @@
+product does not matter, since Odoo uses the routes of the kit’s components for replenishment
+purposes. All other parameters for the kit product may be modified according to preference.
+Finally, click :guilabel:`Save`.
 
-.. image:: kit_shipping/kit-so-line.png
-    :align: center
+The kit’s components require no specific configuration. 
 
-.. image:: kit_shipping/kit-do-picking.png
-    :align: center
+Setting up the kit BoM
+======================
 
-Configuration
-=============
+After fully configuring the kit product and its components, create a BoM for the kit product. Go to
+:menuselection:`Manufacturing --> Products --> Bills of Materials`, then click :guilabel:`Create`.
+Next, set the :guilabel:`BoM Type` to :guilabel:`Kit`. Finally, add each component and specify its
+quantity. Make sure to :guilabel:`Save` the changes.
 
-From the *Products menu* in either the *Inventory* or *Manufacturing* app, create each component 
-product (as you would with any other product), then create the top-level, or kit product. Because
-you cannot track the stock of kit products, the *Product Type* should be set to *Consumable*. 
-However, if you are using Anglo-Saxon accounting, and you would like COGS to be posted when 
-invoicing your kits (and only for this reason), the storable product type should selected instead. 
-Because a kit product cannot be purchased, *Can be Purchased* should be unchecked. The route of the
-kit product in the *Inventory* Tab does not really matter, as, for replenishment, only the route of 
-the components that will be taken into account only. 
+.. image:: kit_shipping/bom-kit-selection.png
+   :align: center
+   :alt: Kit selection on the bill of materials. 
 
-All other parameters on the kit product may be modified according to your preference. The component 
-products require no particular configuration.
+If the kit is solely being used as a sellable product, then only components need to be added, and
+configuring manufacturing operations is not necessary.
 
-Once the products are configured, create a *bill of materials* for the kit product. Add each component
-and its quantity. Select the *BoM Type* Kit. All other options may be left with their default values.
+.. note:: When a kit is sold as a product, it appears as a single line item on the quotation and
+    sales order. However, on delivery orders, each component of the kit is listed. 
 
-.. image:: kit_shipping/kit-bom.png
-    :align: center
+Using kits to manage complex BoMs 
+=================================
 
-Manage Stock of Kit Product and Component Products
-==================================================
+Kits are also used to manage *multilevel BoMs*. These are products that contain other products as
+components and therefore require nested BoMs. Incorporating pre-configured kits into multilevel BoMs
+allows for cleaner organization. List a kit as a component in a higher-level product’s BoM to
```

Comment:
- Add `` Under the :guilabel:`Components`, `` before `List a kit...`
- Uncapitalize the first letter of `List`
- Replace `product's BoM` with `` product's :abbr:`BoM (Bill of Materials)` ``